src/js/theme.ts

```diff
@@ -62,4 +60,5 @@ export default {
   initResponsiveToggler,
   initQuickview,
   initProductBehavior,
+  selectorsMap,
```

Comment:
@NeOMakinG , was it that you wanted ? Shouldn't I rename `selectorsMap` to `selectors` ?
Is it thanks to this export default that we can `const {selectors} = Theme;` ?

Reply:
Yep, rename it to `selectors` (don't forget to rename the import

Yep, this will be naturally exported to `window.Theme` by webpack, check the webpack configuration, you will be able to find the magic